phone_iso3166/e212.py

```diff
@@ -84,6 +57,7 @@
     284: "BG",
     286: "TR",
     288: "FO",
+    289: "GE-AB",
```

Comment:
Since the library is providing 2-letter country code from ISO 3166-1 we cannot support.

Reply:
Yeah, makes sense